Week7/app/src/androidTest/java/com/mmoreno/pokeapp/model/PokeRepositoryTest.kt

```diff
@@ -0,0 +1,114 @@
+package com.mmoreno.pokeapp.model
+
+import androidx.room.Room
+import androidx.test.platform.app.InstrumentationRegistry
+import com.mmoreno.pokeapp.model.dao.PokeDao
+import kotlinx.coroutines.runBlocking
+import org.junit.After
+import org.junit.Assert.assertNotEquals
+import org.junit.Assert.assertNull
+import org.junit.Before
+import org.junit.Test
+
+/**
+ * Implemented class for testing create, update, delete
+ * operations in room database
+ */
+class PokeRepositoryTest {
```

Comment:
Really nice work on these tests, very cool. The assignment mentioned unit tests for the viewmodel, but I'm glad you've included some tests for the database.